packages/zoe/src/contractFacet/types.js

```diff
@@ -0,0 +1,174 @@
+// eslint-disable-next-line spaced-comment
+/// <reference types="ses"/>
+
+/**
+ * @typedef {Object} ContractFacet
+ *
+ * The Zoe interface specific to a contract instance. The Zoe Contract
+ * Facet is an API object used by running contract instances to access
+ * the Zoe state for that instance. The Zoe Contract Facet is accessed
+ * synchronously from within the contract, and usually is referred to
+ * in code as zcf.
+ *
+ * @property {Reallocate} reallocate - reallocate amounts among seats
+ * @property {(keyword: Keyword) => void} assertUniqueKeyword - check
+ * whether a keyword is valid and unique and could be added in
+ * `saveIssuer`
+ * @property {SaveIssuer} saveIssuer - save an issuer to ZCF and Zoe
+ * and get the amountMath and brand synchronously accessible after
+ * saving
+ * @property {MakeInvitation} makeInvitation
+ * @property {() => void} shutdown
+ * @property {() => ZoeService} getZoeService
+ * @property {() => Issuer} getInvitationIssuer
+ * @property {() => Terms } getTerms
+ * @property {(issuer: Issuer) => Brand} getBrandForIssuer
+ * @property {(brand: Brand) => Issuer} getIssuerForBrand
+ * @property {GetAmountMath} getAmountMath
+ * @property {MakeZCFMint} makeZCFMint
+ * @property {(exit: ExitRule= }) => ZcfSeatKit} makeEmptySeatKit
```

Comment:
This feels really picky, but how much does lint care about trailing white space within the braces? I see that `Terms` and `ExitRule=` are followed by a blank before the close brace.  It's a little visually distracting.

Reply:
```js
 * @property {(exit: ExitRule= }) => ZcfSeatKit} makeEmptySeatKit
```
mismatched brackets?